arangod/Network/ConnectionPool.cpp

```diff
@@ -110,18 +108,10 @@ void ConnectionPool::pruneConnections() {
       if ((*it)->fuerte->state() == fuerte::Connection::State::Closed) {
         // lets not keep around disconnected fuerte connection objects
         remove = true;
-      } else {
-        // connection has not yet failed
-        if ((*it)->leases.load() > 0 || (*it)->fuerte->requestsLeft() > 0) {  // continuously update lastUsed
-          (*it)->lastLeased = now;
-          // connection will be kept
-        } else if (aliveCount >= _config.minOpenConnections &&
-                   (now - (*it)->lastLeased) > ttl) {
-          // connection hasn't been used for a while, remove it
-          // if we still have enough others
-          remove = true;
-        } else if (aliveCount >= _config.maxOpenConnections) {
-          // remove superfluous connections
+      } else if ((*it)->leases.load() == 0 && (*it)->fuerte->requestsLeft() == 0) {
```

Comment:
The old code did bump `lastLeased` in this loop for all connections that were still in use.
The new code doesn't do this anymore. Is this going to cause any problems?

Reply:
no